pixi.toml

```diff
@@ -174,25 +174,27 @@ MOZ_HEADLESS = "1"
 PANEL_IPYWIDGET = "1"
 
 [feature.doc.dependencies]
+firefox = "*"
+geckodriver = "*"
```

Comment:
Your issue was a duplicate of https://github.com/holoviz/panel/issues/7119, which discusses adding these dependencies.  

I don't have any strong opinion about it, but I at least want to mention it.

Reply:
I say we add a setting to nbsite to allow you disable the auto-thumbnails and then simply issue warnings for missing thumbnails.